src/ui/DynamicHierarchicalFacetSearch/DynamicHierarchicalFacetSearchValueRenderer.ts

```diff
@@ -102,6 +111,12 @@ export class DynamicHierarchicalFacetSearchValueRenderer {
     return element;
   }
 
+  private renderPathPrefix() {
+    const element = $$('li', { className: PATH_PREFIX_CLASSNAME });
+    element.text(`${l('HierarchicalFacetValuePathPrefix')} `);
```

Comment:
It would be more flexible if we passed the text to the localized label as an argument and use a placeholder in the strings.json (`{0}`). Some languages are written right to left. For these languages, the "in" would be to the right of the search text.